pvm.py

```diff
@@ -43,9 +43,13 @@ def parse_commands(*args):
 		pass
 	# TODO: Create another python file to control two display
 	if command=="file":
-		_logger.info("File set: %s", PEFIX_PATH + value)
+		_logger.info("File set: %s", PREFIX_PATH + value)
+		if IS_FILE_SET:
+			_logger.info("The file has already been set!")
+			if media is not None:
+					media.stop()
 		IS_FILE_SET = True
-		media = OMXPlayer(PEFIX_PATH + value, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--loop'])
+		media = OMXPlayer(PREFIX_PATH + value, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--win', "50,50,1390,1030"])
 		media.pause()
```

Comment:
maybe here, as I was saying, and with the new `VIDEO_PATH`. Or simply just print `value` because we might only be interested in knowing which file, and not the path? Your choice
```suggestion
		media.pause()
		_logger.info("File set: %s", VIDEO_PATH)
```

Reply:
Changed.